### DISCUSSION TITLE
Problem overriding default form plugin styling

### DISCUSSION THREAD

**Discussion by digitalronin** [tailwindlabs/tailwindcss#8600] [Help] [answered]
I've been working through the excellent video tutorials, including [this one about the forms plugin](https://youtu.be/pONeWAzDsQg?t=890).

Everything works fine until I try to override the default form stylings by adding a `@layer base { .. @apply ... }` section to my CSS file, as described.

Nothing I put in there seems to be applied to my HTML, even though everything else works just like it does in the video.

I've put together a [minimal repo. here](https://github.com/digitalronin/tailwindcss-issue#readme), to demonstrate the problem. 

I'm sure I'm missing something obvious, but I can't figure out what it is, so if anyone could point me in the right direction I'd be grateful.

**Comment by jakemulley** [comment, marked as answer]
Have opened a PR on [your repository](https://github.com/digitalronin/tailwindcss-issue/pull/1) @digitalronin - I fell foul of this the other day too. You’ll need to specify an input file else `@tailwindcss/tailwindcss` uses a [fallback](https://github.com/tailwindlabs/tailwindcss/blob/0d064ea032373223922cb17a8b7ece3b3a23c720/src/cli.js#L679) to the standard configuration 👍🏼

**Reply by digitalronin** [reply]
Hey Jake!

Many thanks for that - exactly what I needed.
